### ISSUE TITLE
Convert list of arguments to objects in functions

### ISSUE THREAD

**Issue by ericboucher** [aqualinkorg/aqualink-app#472]
We should standardize our code and update functions that take a list of arguments as input to accept an object with names arguments instead. This helps with readability, maintainability and prevents ordering errors

**Comment by alexmayberry** [comment]
@ericboucher I would love to begin contributing to the project. Is this, as the tag suggests, a good first issue to work on?

**Comment by ericboucher** [comment]
@alexmayberry yes I think it should be! Let me know if you need any help. You can ping me at `eric     @   ovio.org`.
Thanks for proposing your help.

- Eric

**Comment by vr-varad** [comment]
hey @ericboucher I would like to work on this issue if it open for contribution

**Comment by ericboucher** [comment]
Hi @vr-varad, please go for it :)
